src/test/resources/testng.xml

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
+<suite name="MyTestSuite">
+    <test name="Test1" parallel="methods">
```

Comment:
add <suite name="MyTestSuite" parallel="methods" thread-count="2">
Limit the number of simultaneous threads